doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.rst

```diff
@@ -32,11 +33,16 @@ Steps
 
    .. code-block:: bash
 
-    DOCKER_IMAGE=rolling-tutorial docker compose run --rm --name moveit2_container gpu
+    DOCKER_IMAGE=main-rolling-tutorial-source docker compose run --rm --name moveit2_container gpu
 
-   You can replace ``rolling-tutorial`` with other tagged images, e.g. ``humble-tutorial``. Similarly, you can replace ``gpu`` with ``cpu`` if you do not wish to run using Nvidia GPU drivers and you can change the name of the container by replacing ``moveit2_container``. The ``--rm`` argument will remove the container when you stop (or exit) it, otherwise you can keep your modified container on disk and start it using ``docker start moveit2_container``
+   You can replace ``DOCKER_IMAGE`` with other Docker image names, such as ``main-jazzy-tutorial-source`` or ``humble-humble-tutorial source``.
+   This is based on the convention ``<MoveIt branch name>-<ROS distro name>-tutorial-source``.
 
-4. You should now be inside of your Docker container, in the workspace directory, with the completed :doc:`Planning Around Objects </doc/tutorials/planning_around_objects/planning_around_objects>` and :doc:`Pick and Place with MoveIt Task Constructor </doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor>` tutorials. Go ahead and try one of the launch commands like ``ros2 launch moveit2_tutorials demo.launch.py``
+   Similarly, you can replace the ``gpu`` service name with ``cpu`` if you do not wish to run using Nvidia GPU drivers and you can change the name of the container by replacing ``moveit2_container``.
```

Comment:
```suggestion
   Similarly, you can replace the ``gpu`` service name with ``cpu`` if you do not wish to run using NVIDIA GPU drivers and you can change the name of the container by replacing ``moveit2_container``.
```
Not to be a brand guideline shill, but I noticed you made a change earlier in this doc to the CAPSLOCK version, so just trying to keep consistentcy.

Reply:
Thank you! I changed one of them but my pedantry lost to my microscopic attention span.